src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2292,20 +2292,17 @@ internal IOperation CreatePropertySubpatternMember(Symbol? symbol, ITypeSymbol m
         {
             var nameSyntax = (syntax is SubpatternSyntax subpatSyntax ? subpatSyntax.NameColon?.Name : null) ?? syntax;
             bool isImplicit = nameSyntax == syntax;
+            var receiver = symbol?.IsStatic == false ? new InstanceReferenceOperation(InstanceReferenceKind.PatternInput, _semanticModel, nameSyntax, matchedType, isImplicit: true) : null;
```

Comment:
>receiver [](start = 16, length = 8)

Can we extract this to a local function instead? We don't need to create a receiver in the `default:` case.